staging/src/k8s.io/apiserver/pkg/server/filters/priority-and-fairness_test.go

```diff
@@ -696,8 +695,12 @@ func TestPriorityAndFairnessWithPanicRecoveryAndTimeoutFilter(t *testing.T) {
 		)
 
 		apfConfiguration := newConfiguration(fsName, plName, userName, plConcurrencyShares, 0)
-		stopCh := make(chan struct{})
-		controller, controllerCompletedCh := startAPFController(t, stopCh, apfConfiguration, serverConcurrency, plName, plConcurrency)
+		ctx, cancel := context.WithCancel(context.Background())
```

Comment:
In unit tests use `_, ctx := ktesting.NewTestContext(t)` to get per-test output.

Reply:
it's in staging. So we cannot use it, right? Same for those below.

Reply:
We can use `k8s.io/klog/v2/ktesting`. It does the per-test output, which is better than nothing.

Reply:
I like @pohly 's suggestion. Everywhere it applies.

Reply:
Oh, didn‘t notice it‘s in klog 👍

Reply:
Resolved, AFAIC